litmus-portal/backend/auth/pkg/utils/invite/invite.go

```diff
@@ -0,0 +1,81 @@
+package invite
+
+import (
+	"bytes"
+	"html/template"
+	"log"
+	"net/smtp"
+	"os"
+)
+
+//MIME
+const (
+	MIME = "MIME-version: 1.0;\nContent-Type: text/HTML; charset=\"UTF-8\";\n\n"
+)
+
+//Request struct
+type Request struct {
+	from    string
+	to      []string
+	subject string
+	body    string
+}
+
+//NewRequest creates a request
+func NewRequest(to []string, subject string) *Request {
+	return &Request{
+		to:      to,
+		subject: subject,
+	}
+}
+
+//ParseTemplate parses template file and executes it with the template data supplied
+func (r *Request) ParseTemplate(templateFileName string, data interface{}) error {
+	t, err := template.ParseFiles(templateFileName)
+	if err != nil {
+		return err
+	}
+	buf := new(bytes.Buffer)
+	if err = t.Execute(buf, data); err != nil {
+		return err
+	}
+	r.body = buf.String()
+	return nil
+}
+
+//Sendmail parses the HTML template that is sent in the email body and calls SendEmail()
+func Sendmail(recipients []string, username string, password string) {
+
+	templateData := struct {
+		Username string
+		Password string
+	}{
+		Username: username,
+		Password: password,
+	}
+	r := NewRequest(recipients, "Litmus Portal")
+	if err := r.ParseTemplate("/home/saranyaj/MayaData/my work(active)/litmus/litmus-portal/backend/auth/pkg/utils/invite/templates/template.html", templateData); err == nil {
```

Comment:
can you change this parse template directory to something more generic? cc: @rajdas98 @Sanjay1611 @gdsoumya